src/shared/codelens/samTemplateCodeLensProvider.ts

```diff
@@ -6,13 +6,14 @@
 import * as _ from 'lodash'
 import * as vscode from 'vscode'
 import { TemplateFunctionResource, TemplateSymbolResolver } from '../cloudformation/templateSymbolResolver'
-import { getReferencedTemplateResources, LaunchConfiguration } from '../debug/launchConfiguration'
-import { TEMPLATE_TARGET_TYPE } from '../sam/debugger/awsSamDebugConfiguration'
+import { getConfigsMappedToTemplates, LaunchConfiguration } from '../debug/launchConfiguration'
+import { TEMPLATE_TARGET_TYPE, API_TARGET_TYPE } from '../sam/debugger/awsSamDebugConfiguration'
 import { AddSamDebugConfigurationInput } from '../sam/debugger/commands/addSamDebugConfiguration'
 import { localize } from '../utilities/vsCodeUtils'
 
 /**
- * Provides Code Lenses for generating debug configurations for SAM templates.
+ * Provides "Add Debug Configuration" CodeLenses to SAM template.yaml files,
```

Comment:
Thoughts about using alternate text for the API based CodeLenses that indicate they are API related?

Reply:
I considered it but I don't see a strong case for it. The codelens is attached to the node it operates on, which strongly implies that the action is particular to that context.

Reply:
Will it be clear to the user that the nested `Add Debug Configuration` behaves differently than the top-level one? If this wasn't nested I wouldn't be concerned about the same wording.

Reply:
Maybe? Does this need to block this PR...?

Reply:
Renamed to `Add API Debug Configuration`